test/e2e/tests/editor/editor.pictures.spec.ts

```diff
@@ -76,7 +74,11 @@ test.describe('Editor pictures', () => {
       await expect(caption).toBeVisible(); // it disappears immediately which could be annoying
       await caption.fill('');
       await expect(caption).not.toBeVisible(); // it disappears immediately which could be annoying
+      // Navigate away to force changes to be saved
```

Comment:
Did you try just doing something like `caption.blur()`? (I'm not sure if that method exists, but surely there's an API somewhere that'll do it)

Reply:
There is a `locator.blur()` method, and I didn't think to try that. I'll try it and see if that's enough.

Reply:
Interestingly, doing `caption.blur()` immediately after `caption.fill('')` caused the test to fail because "Error: locator.blur: Target page, context or browser has been closed". Which I think is incorrect; I think the actual problem is that as soon as you empty the caption, it gets hidden, so `.blur()` suddenly can't find the locator. I'll try again with `if (caption.isVisible()) { caption.blur() }` and see what that does.

Reply:
Well, `caption.blur()` didn't help. Probably because the caption immediately goes invisible and so the Playwright locator can't call `.blur()` on it, and so Firefox isn't sending the blur event. I'll try one more thing: calling .focus() on a different locator, which should cause the blur event to be fired on the caption. If that fails I'll stick with the code as-is.

Reply:
Calling `.focus()` on the lexeme field didn't work on Firefox either. I'm going to leave this as-is, because I have not gotten the simpler alternatives to work and this is simple enough, all things considered.

Reply:
@myieye - Please feel free to resolve this conversation if you're satisfied. I'm leaving it open so it's easier for you to find when you do a re-review.